microsoft/botframework-components
# chore: Merge latest main into feature/telephony

### Purpose

Merge latest main into feature/telephony

### Changes

- git revert dfedd60 // revert the previous merge
- git merge main --no-ff // Merge the latest main
- Fix Microsoft.Bot.Components.sln and Microsoft.Bot.Components.Tests.sln manually since Telephony projects are in the older test folders

### Tests

- Ensure Microsoft.Bot.Components.sln and Microsoft.Bot.Components.Tests.sln build
- Ensure all tests in Microsoft.Bot.Components.Tests.sln run successfully
- Compare this branch and ensure no diffs other than new telephony

TODO: Ensure merge is not a squash merge to preserve history\

#minor
